packages/plugin-dts/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "rsbuild-plugin-dts",
   "version": "0.0.14",
-  "description": "Dts plugin for Rsbuild",
+  "description": "Rsbuild plugin that provides support for emitting declaration files for Typescript in Rslib",
```

```suggestion
  "description": "Rsbuild plugin that supports emitting declaration files for TypeScript in Rslib.",
```

And, does it implicit it only works with Rslib?

---

It works in Rsbuild project, may be we can delete `in Rslib`.